--- rsnapshot-1.5.1/rsnapshot-program.pl	2025-01-27 03:00:25.000000000 +0100
+++ rsnapshot-1.5.1/rsnapshot-program.pl	2026-08-03 03:16:58.319908726 +0200
@@ -7221,6 +7221,8 @@
 
 =back
 
+=for comment OpenMandriva: keep the extra =back so the outer list is closed for pod2man
+
 Remember that tabs must separate all elements, and that
 there must be a trailing slash on the end of every directory.
 
